### ISSUE TITLE
Clicking on the icon in the dock does not "show" when "hidden"

### ISSUE THREAD

**Issue by kikorb** [KRTirtho/spotube#1210]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The app is not shown, you need to click on the systray show/hide toggle .

### Expected Behavior

The application is shown again

### Steps to reproduce

Open Spotube and hide the application either
1. using the x button to close the window
2. using the hide option in the systray

Clicking on the dock icon in macos usually shows the applications previously hidden.

### Operating System

macOS

### Spotube version

3.4.1 (28)

### Installation source

GitHub Releases (Binary)

### Additional information

If you use Cmd+H to hide the window and then click on the dock icon the application DOES show

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by AuburnMedia** [comment]
I agree this is annoying, but it is simply fixed by terminal integration.

It seems that osascript provides a simple solution to this, here is a rough example:
`osascript -e 'tell application "spotube"' -e 'set visible of process "Twitter" to false' -e 'end tell'`

Working on implementing this 😉

**Comment by AuburnMedia** [comment]
I found a simple implementation in AppleScript.

Here is a rough example:
`osascript -e 'tell application "Finder"' -e 'set visible of process "spotube" to false' -e 'end tell'`

It seems to be a bit slow to run, slower than you'd expect, but I'm working on it.
